yarn-error.log

```diff
@@ -0,0 +1,1822 @@
+Arguments: 
```

Comment:
this file shouldn't be checked in.  maybe you can update the git-ignore to exclude this error log?